R/mmrm-package.R

```diff
@@ -8,12 +8,13 @@
 #' @useDynLib mmrm, .registration = TRUE
 #' @importFrom Rcpp evalCpp
 #' @import checkmate
-#' @importFrom lifecycle deprecated
+#' @importFrom lifecycle deprecated badge
```

Comment:
please just import one function from each package, because we directly access them from each package's namespace. Example:
```suggestion
#' @importFrom lifecycle deprecated
```
because we use `lifecycle::badge()` where needed

Reply:
removed